package.json

```diff
@@ -80,10 +80,8 @@
     "@typescript-eslint/eslint-plugin": "^7.0.0",
     "@typescript-eslint/parser": "^6.21.0",
     "@vitest/coverage-istanbul": "2.1.8",
-    "esbuild-jest": "^0.5.0",
-    "esbuild": "^0.24.2",
```

Comment:
These aren't used (as far as I can tell!).

The build works without them - it's likely they should have been removed when we swapped out jest for vite.